src/Aeternity/Address.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Address.h"
+#include "Identifiers.h"
+#include <Base58.h>
+#include <Coin.h>
+#include <HexCoding.h>
+
+using namespace TW::Aeternity;
+
+/// Determines whether a string makes a valid address.
+bool TW::Aeternity::Address::isValid(const std::string &string) {
+    if (string.empty()) {
+        return false;
+    } else {
```

Comment:
we can move the logic out of the `else` clause, it will early return if string is empty